introduction.md

```diff
@@ -1,3 +1,5 @@
 Hello all, I am Jake Rushing from Dice.com and I am taking this course to better myself in git. I hope that you think I'm awesome! :D
 
-My favorite path so far is Javascript!
\ No newline at end of file
+*Javascript
+*Git
+*HTML/CSS
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?